test/resqml2_0_1test/AbstractBigIjkGridRepresentationTest.cpp

```diff
@@ -0,0 +1,223 @@
+#include "resqml2_0_1test/AbstractBigIjkGridRepresentationTest.h"
+
+#include "catch.hpp"
+#include "resqml2_0_1test/LocalDepth3dCrsTest.h"
+
+#include "resqml2_0_1/LocalDepth3dCrs.h"
+#include "resqml2_0_1/IjkGridExplicitRepresentation.h"
+#include "resqml2_0_1/DiscreteProperty.h"
+#include "resqml2_0_1/ContinuousProperty.h"
+
+using namespace std;
+using namespace common;
+using namespace resqml2_0_1test;
+using namespace resqml2;
+
+AbstractBigIjkGridRepresentationTest::AbstractBigIjkGridRepresentationTest(
+	const string & epcDocPath,
+	const unsigned int & iCount, const unsigned int & jCount, const unsigned int & kCount,
+	const unsigned int & faultCount,
+	const double & xMin, const double & xMax, const double & yMin, const double & yMax, const double & zMin, const double & zMax,
+	const double & faultThrow, 
+	const char * defaultUuid, const char * defaultTitle)
+	: iCount(iCount), jCount(jCount), kCount(kCount), faultCount(faultCount), 
+	xMin(xMin), xMax(xMax), yMin(yMin), yMax(yMax), zMin(zMin), zMax(zMax), faultThrow(faultThrow),
+	AbstractIjkGridRepresentationTest(
+		epcDocPath, defaultUuid, defaultTitle, 
+		initNodesCountIjkGridRepresentation(iCount, jCount, kCount, faultCount),
+		initNodesIjkGridRepresentation(iCount, jCount, kCount, faultCount, xMin, xMax, yMin, yMax, zMin, zMax, faultThrow)) {
+	if ((iCount < 1) || (jCount < 1) || (kCount < 1))
+	{
+		throw invalid_argument("iCount, jCount and kCount must be >= 1.");
+	}
+
+	if (faultCount > (iCount - 1))
+	{
+		throw invalid_argument("faultCount must be strictly lesser than iCount.");
```

Comment:
less vs lesser : https://english.stackexchange.com/questions/24719/difference-between-less-and-lesser
I think it should be "less" in this case